### PR TITLE
Update "Display Settings" app strings to "Settings"

### PR DESCRIPTION
**Pull request by kofi-q**
## Overview

Since we're starting to include audio settings in the formerly "Display Settings" screen, this renames the screen title and button label accordingly, along with naming in the code, for consistency. We'll probably also need to update the button icon with something more general than the screen icon, but will save that for later.

## Demo Video or Screenshot

https://github.com/votingworks/vxsuite/assets/264902/578b9554-a77a-48a8-a7e8-76e2e9f8d8dd

## Testing Plan
- Updated existing tests + manual verification

## Checklist

- ~[ ] I have added [logging](https://github.com/votingworks/vxsuite/tree/main/libs/logging) where appropriate to any new user actions, system updates such as file reads or storage writes, or errors introduced.~

- [x] I have added a screenshot and/or video to this PR to demo the change
- ~[ ] I have added the "user_facing_change" label to this PR to automate an announcement in #machine-product-updates~

### CODE REVIEW

File: apps/scan/frontend/src/components/display_settings_button.tsx [hunk 1]
```diff
@@ -21,10 +21,13 @@ export function DisplaySettingsButton(): JSX.Element | null {
   }
 
   return (
-    <Button onPress={history.push} value={Paths.DISPLAY_SETTINGS}>
```
Comment (kofi-q) [inline apps/scan/frontend/src/components/display_settings_button.tsx:24]: fixing broken `onPress` handler here - the `Button` was recently changed to fire `onPress` with two arguments instead of one

File: libs/ui/src/display_settings/tab_bar.tsx [hunk 2]
```diff
@@ -23,7 +24,7 @@ const Container = styled.div<ContainerProps>`
   gap: max(0.25rem, ${(p) => p.theme.sizes.minTouchAreaSeparationPx}px);
 
   & > * {
-    max-width: ${100 / PANE_IDS.length}%;
+    max-width: ${(p) => 100 / p.tabCount}%;
```
Comment (kofi-q) [inline libs/ui/src/display_settings/tab_bar.tsx:27]: unrelated fix for tabs getting unnecessarily shortened when there are only 2 tabs displayed

**Review by arsalansufi — APPROVED** [review]
